controllers/workspace/condition.go

```diff
@@ -0,0 +1,65 @@
+//
+// Copyright (c) 2019-2021 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+
+package controllers
+
+import (
+	dw "github.com/devfile/api/v2/pkg/apis/workspaces/v1alpha2"
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	PullSecretsReadyCondition dw.WorkspaceConditionType = "PullSecretsReady"
```

Comment:
If we try to align conditions names, it should be
```suggestion
	PullSecretsReady dw.WorkspaceConditionType = "PullSecretsReady"
```